learing_journal/tests.py

```diff
@@ -93,7 +94,7 @@ def test_lists_title(new_session):
 def test_lists_body(new_session):
     '''tests wether list() pull out correct data from db'''
     from .views.default import home
-    new_session.add(MyModel(title='test1', body='test2', date='test3'))
+    new_session.add(Journal(title='test1', body='test2', date=datetime.datetime.now()))
     new_session.flush()
     result = home(dummy_http_request(new_session))
     for entry in result['entries']:
```

Comment:
This is effectively a rehash of the previous test. You probably don't need this one.